src/server/myCoolService/myCoolService.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
+
+# Yeah I did it.  I wrote my service in bash!
+
+
+
+echo "Your are awesome!"
+
+
+
+
+
+echo "Done!"
```

Comment:
I wonder what this will look like when another user comes along and tries to add a comment here.   I wonder what type of control we have on comments in general.  I wonder if I can filter comments ..  How would I see comments that are made after this comment.  Can I collapses comments ... hummm